elementary/calendar
# Use the same method to strip time everywhere

Adding a new event in Calendar with a begin time and end time in the same day caused the event not to show up on the sidebar "agenda".

I found that the AgendaView widget was not using the same `strip_time ()` function as other parts of the code, and that the `strip_time ()` function needed to create a new `DateTime.utc ()` value in order to function correctly throughout the program.

Test A:
1. Create a new event with a start and end time within the same day
2. Verify that the event shows up on the correct day in the main grid
3. Select that day in the main calendar grid
4. Verify that the event shows up in the sidebar with the correct start and end time

Test B:
1. Create a new event with a start time on one day, and end time on the following day
2. Verify that the event shows up on the two correct days in the main grid
3. Select the first day in the main calendar grid
4. Verify that the event shows up in the sidebar with the correct start and end date/time
5. Select the second day in the main calendar grid
6. Verify that the event shows up in the sidebar with the correct start and end date/time

Test C:
1. Create a new _all day_ event spanning 3 days
2. Verify that the event shows up on the three correct days in the main grid
3. Select each day in the main grid, verifying that the event shows up in the Sidebar for all three days
